### ISSUE TITLE
Network Error 413

### ISSUE THREAD

**Issue by plasmawario** [beat-machine/beat-webapp#41]
Recently came across this website and i absolutely love it! Just wanted to share a strange thing that's been happening: Seems like certain songs i'm attempting to upload end up returning a 413 network error. I've been uploading music from the Distance OST and some of them work while some don't. 

Here are the ones i've tried that do work:
Terminus, Euphoria, Ghost, An Imperfect Rhythm, Embers

Here's some of the ones i've tried that return the error:
Compression, Cataclysm, Aftermath, Grime

What confuses me is the File Size, like the error code suggests, as Terminus (8.69MB) has a larger file size than Aftermath (7.46MB), though the rest of the files i have uploaded and had success with were below 8MBs

**Comment by wileyfoxyx** [comment]
Yeah, it seems that the problrem is with file size. I tried to upload .mp3 file with size a bit over 9MB and got stuck with this error too. Compressing file helped though.

**Comment by branchpanic** [comment]
Yeah, 413 is returned when a file is greater than around 8 MB. This isn't really communicated on the site.

For now, I made it more lenient. The site should now accept files that are a bit over 8 MB. In your case, hopefully it won't incorrectly reject Aftermath anymore.

If you still have problems feel free to continue the thread or try compressing the file as wileyfoxyx suggested. 

Obviously this isn't the greatest user experience and is something I'd improve if I came back to this project.
